### ISSUE TITLE
Coloring of ( ) doesn't work on latex files

### ISSUE THREAD

**Issue by kareltdlg** [CoenraadS/BracketPair#132]
Missing coloring of ( ) in .tex files.
![image](https://user-images.githubusercontent.com/30301939/40270912-6b9e345a-5b63-11e8-987d-b3416cc753c5.png)

**Comment by allenryb** [comment]
I come across the same problem. Moreover, Bracket Pair Colorizer fails in math environment.

Is there any solution for them? Thanks!

**Comment by linhtto** [comment]
Yes, I confirm both of these problems. It's especially sad that it doesn't work at all in math environments: this is when these nested brackets show up the most and are most important to keep track of in LaTeX.

**Comment by louie6111** [comment]
Yes. Anyone found the solution?

**Comment by nasosev** [comment]
> 
> 
> Yes. Anyone found the solution?

The 'Rainbow Brackets' extension seems to work fine.

**Comment by louie6111** [comment]
> The 'Rainbow Brackets' extension seems to work fine.

Awesome! Thanks!
